controllers/drcluster_controller.go

```diff
@@ -141,6 +145,24 @@ func (r *DRClusterReconciler) drClusterConfigMapMapFunc(configMap client.Object)
 	return requests
 }
 
+func (r *DRClusterReconciler) drClusterSecretMapFunc(secret client.Object) []reconcile.Request {
+	if secret.GetNamespace() != NamespaceName() {
+		return []reconcile.Request{}
```

Comment:
Why not return nil? (empty slices is nil)

Reply:
I am returning it for the consistency with the other code. The same return can be found in 5 more places in this file.